docs: Loan management module documentation (v13) (#573)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/loan-management/loan-security-type.md

<!-- add-breadcrumbs -->
# Loan Security Type
> Introduced in Version 13

> **Loan security type defines the type of security that can be pledged against a loan. For Eg: Stocks, land, etc.**

Reviewer: ```suggestion
**Loan Security Type defines the type of security that can be pledged against a loan like Stocks, land, etc.**
```

> During a loan process, if the loan for which the applicant is applying is a secured loan then some kind of security needs to be pledged against that loan. In loan security type master you can define that type of security.

Reviewer: ```suggestion
During a loan process, if the loan for which the applicant is applying is a secured loan then some kind of security needs to be pledged against that loan. In Loan Security Type master you can define that type of security.
```

> <img class="screenshot" alt="Make Loan Security Type" src="{{docs_base_url}}/assets/img/loan-management/loan-security-type-flow.png">

Reviewer: arrows!

To access the Loan Security Type list, go to:
> Home > Loan Management > Loan Security > Loan Security Type

> <img class="screenshot" alt="Loan Security Type" src="{{docs_base_url}}/assets/img/loan-management/loan-security-type.png">

Reviewer: screenshot top needs cropping

A Loan Security Type contains details about:

>   * The Unit Of Measurement of that loan security

Reviewer: better formatting needed

  * The Haircut % on that loan security

> > Haircut percentage is the percentage difference between market value of the loan security and the value ascribed to that asset when used as collateral for that loan.

Reviewer: ```suggestion
> Haircut percentage is the percentage difference between market value of the Loan Security and the value ascribed to that asset when used as collateral for that loan.
```
